keyboards/handwired/stream_cheap/2x4/keymaps/via/keymap.c

```diff
@@ -0,0 +1,28 @@
+#include QMK_KEYBOARD_H
+
+
+enum layers {
+  NORMAL_LAYER = 0,
+  SECOND_LAYER = 1,
+  THIRD_LAYER = 2,
+  FOURTH_LAYER = 3
```

Comment:
No need to specify the numbers.  Enums start at 0 and automatically increment up.
```suggestion
    NORMAL_LAYER,
    SECOND_LAYER,
    THIRD_LAYER,
    FOURTH_LAYER,
```

Reply:
Addressed in 9be12f6